modules/virt-customizing-storage-profile-snapshot-class-cli.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/storage/virt-configuring-storage-profile.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-customizing-storage-profile-snapshot-class-cli_{context}"]
+= Specifying a volume snapshot class by using the CLI
+
+If you are creating a snapshot of a VM, a warning appears if the storage class of the disk has more than one volume snapshot class associated with it. In this case, you must specify one volume snapshot class; otherwise, any disk that has more than one volume snapshot class is excluded from the snapshots list.
+
+You can select which volume snapshot class to use by either:
+
+* Setting the `spec.snapshotClass` for the storage profile
+* Setting a default volume snapshot class
+
+.Procedure
+
+* Set the volume snapshot class you want to use, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata:
+  name: ocs-storagecluster-ceph-rbd-virtualization
+spec:
+  snapshotClass: ocs-storagecluster-rbdplugin-snapclass
+----
++
+* Alternatively, set the default volume snapshot class by running the following command:
```

Comment:
This would be better as an optional step.

```suggestion
* Optional: Set the default volume snapshot class by running the following command:
```

Reply:
Making it an optional step seems confusing. It isn't a consecutive step; it's just another way of performing the procedure.